app/assets/stylesheets/variables/typography.scss

```diff
@@ -137,12 +137,19 @@ $font-book: (
     color: white
   ),
   search-result-title: (
-    font-family: Open Sans,
-    font-size: 1.25rem,
-    font-weight: 500,
-    line-height: 1.5,
+    font-family: Merriweather,
+    font-size: 1.375rem,
+    font-weight: 700,
+    line-height: 1.4,
     color: $prussian-blue
   ),
+  search-result-link: (
+    font-family: Open Sans,
```

Comment:
I think we should put font family names with spaces in to single quotes (e.g. 'Open Sans')

Reply:
Just to note, i've already done this for the other fonts on another branch